osx/spec/localhost/android_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe 'android' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.